### PR TITLE
migration: Add case about pre-create target image

### PR DESCRIPTION
**Pull request by cliping**
XXX-297908 - VM live migration with copy storage - don't pre-create target image manually

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

 (1/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_all.raw_format.p2p: PASS (140.41 s)
 (2/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_all.raw_format.non_p2p: PASS (140.85 s)
 (3/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_all.qcow2_format.p2p: PASS (149.75 s)
 (4/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_all.qcow2_format.non_p2p: PASS (143.10 s)
 (5/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_inc.raw_format.p2p: PASS (98.04 s)
 (6/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_inc.raw_format.non_p2p: PASS (98.05 s)
 (7/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_inc.qcow2_format.p2p: PASS (99.89 s)
 (8/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.no_precreate_target_disk.copy_storage_inc.qcow2_format.non_p2p: PASS (100.23 s)

File: libvirt/tests/cfg/migration_with_copy_storage/no_precreate_target_disk.cfg [hunk 1]
```diff
@@ -0,0 +1,49 @@
+- migration_with_copy_storage.no_precreate_target_disk:
+    type = no_precreate_target_disk
```
Comment (chunfuwen) [inline libvirt/tests/cfg/migration_with_copy_storage/no_precreate_target_disk.cfg:2]: no_precreate_target_disk --> precreate_no_target_disk

File: libvirt/tests/cfg/migration_with_copy_storage/no_precreate_target_disk.cfg [hunk 2]
```diff
@@ -0,0 +1,49 @@
+- migration_with_copy_storage.no_precreate_target_disk:
```
Comment (chunfuwen) [inline libvirt/tests/cfg/migration_with_copy_storage/no_precreate_target_disk.cfg:1]: no_precreate_target_disk -->precreate_no_target_disk

Comment (cliping) [reply libvirt/tests/cfg/migration_with_copy_storage/no_precreate_target_disk.cfg]: updated, thanks.

File: libvirt/tests/cfg/migration_with_copy_storage/no_precreate_target_disk.cfg (hunk 1 above)
Comment (cliping) [reply libvirt/tests/cfg/migration_with_copy_storage/no_precreate_target_disk.cfg]: Updated, thanks.

**Review by nanli1 — APPROVED** [review]

**Pull request merged**